lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -67,18 +67,6 @@ def build_index(self, server_config, dates):
         return f"/{index}"
 
     def build_index_from_metadata(self) -> str:
-        """
-            Retrieve the list of ES indices from the dataset index
-            map metadata based on a given root index name.
-
-            Args:
-                root_index_name: root index name
-
-            Returns:
-                An Elasticsearch query URL string listing the set of
-                indices containing documents for the specified root
-                index name.
-        """
```

Comment:
Hmmm...I think having _some_ sort of docstring would be good....  😁

(I only asked for the `Args` section to be removed...I think the rest of it is OK. 🙂)